src/routes.js

```diff
@@ -45,6 +46,13 @@ var getFirstBrowserLanguage = function () {
   return null
 }
 
+var onEnter = function() {
+  let parsedParams = queryString.parse(window.location.hash)
+  let path = JSON.parse(JSON.stringify(parsedParams.state)).slice(13, -2)
```

Comment:
These numbers feel magic -- can you either add a comment explaining why, and/or (preferably) do some parsing which is more robust?  I fear that this code could break mysteriously.